src/components/slide.js

```diff
@@ -64,7 +64,7 @@ class Slide extends React.PureComponent {
               addFragment({
                 className: frag.className || '',
                 slide: this.props.hash,
-                id: `${this.props.slideIndex}-${currentOrder}`,
```

Comment:
this is the key here: we were sometimes referring to fragments by their slide's index and other times by their slide's hash. Usually those two are the same. Until you set a custom slide id. 😄